data/com.endlessm.GoogleChromeInitialSetup.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Name=Google Chrome Initial Setup
+Exec=/usr/bin/eos-google-chrome-installer
```

Comment:
I don't have much knowledge about autotools, but `$(pkglibexecdir)` seems to be `$(prefix)/libexec/<pkgname>`, not `$(prefix)/lib/<arch>/<pkgname>`.